src/widgets/_windows.scss

```diff
@@ -170,10 +170,16 @@ window:not(.popup):not(.menu) {
         background-color: #{"@color_primary"};
 
         .title {
+            font-size: 9pt;
```

Comment:
Is this necessary?

Reply:
It's necessary since label size might change with the user prefence, and enforcing a size using `pt` ensures it scales properly.

Reply:
Can you elaborate on what you mean by this? As far as I can tell, this seems to have no visible effect

Reply:
It's basically "more correct" measurement unit wise. Points are scaled automatically to a whole number of physical pixels in order to accommodate for screen size, pixel density and typical viewing distance, so it means it's the better measurement unit to use.

Reply:
And third party apps might want to change their titles & subtitles sizes, so this ensures the starting point has a default.

Reply:
Can you post a screenshot showing the difference between including this line versus not including it?

Reply:
The changes most visible is with the red highlights in both here:

`no-line`
![no-line](https://user-images.githubusercontent.com/4886639/87482815-9c261180-c608-11ea-88d1-e475a73dd5f7.png)

`with-line`
![with-line](https://user-images.githubusercontent.com/4886639/87482937-e0b1ad00-c608-11ea-8d04-68c9660e0132.png)

Looking at it, it's visible the font also doesn't have wild kerning anymore (check it out with glimpse, make `with-line` semi-opaque above `no-line` and see that both don't match perfectly. The `with-line` looks better then.)